_dashboards/maps/selfhost-maps-server.md

```diff
@@ -1,7 +1,10 @@
 ---
 layout: default
 title: Self-host maps server
```

Comment:
Please change name to match convention: Configuring the self-host maps server?

Reply:
Revised